SocialFilter/sections/specific.txt

```diff
@@ -1664,7 +1664,7 @@ sport.tvp.pl##.article__social > a[class^="article__social-"]
 arabam.com##a[ng-click^="self.socialShareTab"]
 muslimmedya.com###sidebar > #text-7
 metalinjection.net##.sticky_social
-||platform.twitter.com/widgets.js$domain=real-vin.com|womanel.com.ua|tribundergi.com|inforesist.org|cj3z.com|orbita.dn.ua|ppt.ru|obltv.ru|anilibria.tv|eurolab.ua|anidub.com|losst.ru|cwbaltimore.com|fox38corpuschristi.com|fox47.com|foxkansas.com|foxrichmond.com|kbsi23.com|kdsm17.com|raleighcw.com|star64.tv|wcti12.com|wpgh53.com|wtto21.com|wutv29.com|ellegirl.ru|mforos.com|news-journalonline.com|insaatnoktasi.com|pctip.ru|nk.org.ua|gercekizmir.com|xn----8sbcm1bmcar0m.xn--p1ai
+||platform.twitter.com/widgets.js$domain=real-vin.com|womanel.com.ua|tribundergi.com|inforesist.org|cj3z.com|orbita.dn.ua|ppt.ru|obltv.ru|anilibria.tv|eurolab.ua|anidub.com|losst.ru|cwbaltimore.com|fox38corpuschristi.com|fox47.com|foxkansas.com|foxrichmond.com|kbsi23.com|kdsm17.com|raleighcw.com|star64.tv|wcti12.com|wpgh53.com|wtto21.com|wutv29.com|ellegirl.ru|mforos.com|news-journalonline.com|insaatnoktasi.com|pctip.ru|nk.org.ua|gercekizmir.com|xn----8sbcm1bmcar0m.xn--p1ai|altyazi.net
```

Comment:
Please put new domains at the front of domain list for new rules, thank you.

Reply:
no. New domains should be added at the and of URL blocking rules, and at the front of elemhide, js etc.(except groups of rules, for example anti-BAB)

Reply:
if you add in front or random place, review will take more time

Reply:
Random place is normal it takes longer, but I see your point to the end in case block rule is very long.
Imho it is just a bit strange for non-block rules front, others end.

Reply:
What strange? If end or start of the `line` is changed, you will see this change immediately - there are only two cases - start and end of the line.
For example:
`500 symbols rule`+`modifier with 100500 domains`
if you add new domain  to the start of domains list, you will spend a lot of time to find what was changed
The same with 
`100500 domains`+`100500 symbols JS rule` and adding domain to not first position

But when you change a group of rules(for example anti-adblock rules for different products) better add the rules to start. For example
```
/wp-content/uploads/*/*.js$domain=haohaiyo.cc|51kkp.club|infot3chpro.com|[500 domains]|gizchina.com
haohaiyo.cc,51kkp.club,infot3chpro.com,[500 domains],gizchina.com$$script...
```

Reply:
This is important when you resolve merging conflicts. In other case you can lose previous changes.